[2021-01-10 01:20:48,787] {_internal.py:113} MainThread INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-10 01:20:48,788] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-10 01:20:49,873] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-10 01:20:49,874] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 339-380-536
[2021-01-10 01:20:57,155] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [10/Jan/2021 01:20:57] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-10 01:21:53,986] {_internal.py:113} MainThread INFO in _internal:  * Detected change in '/home/user2/Desktop/DATA_PIPELINE (copy)/resources/data_pipeline_resources.py', reloading
[2021-01-10 01:21:54,122] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-10 01:21:55,253] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-10 01:21:55,254] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 339-380-536
[2021-01-10 01:22:49,183] {_internal.py:113} MainThread INFO in _internal:  * Detected change in '/home/user2/Desktop/DATA_PIPELINE (copy)/resources/data_pipeline_resources.py', reloading
[2021-01-10 01:22:49,304] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-10 01:22:50,175] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-10 01:22:50,176] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 339-380-536
[2021-01-10 01:22:59,336] {_internal.py:113} MainThread INFO in _internal:  * Detected change in '/home/user2/Desktop/DATA_PIPELINE (copy)/resources/data_pipeline_resources.py', reloading
[2021-01-10 01:22:59,474] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-10 01:23:00,248] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-10 01:23:00,249] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 339-380-536
[2021-01-10 01:23:08,388] {_internal.py:113} MainThread INFO in _internal:  * Detected change in '/home/user2/Desktop/DATA_PIPELINE (copy)/resources/data_pipeline_resources.py', reloading
[2021-01-10 01:23:08,531] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-10 01:23:09,447] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-10 01:23:09,447] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 339-380-536
[2021-01-10 01:23:20,652] {_internal.py:113} MainThread INFO in _internal:  * Detected change in '/home/user2/Desktop/DATA_PIPELINE (copy)/resources/data_pipeline_resources.py', reloading
[2021-01-10 01:23:20,772] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-10 01:23:21,614] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-10 01:23:21,615] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 339-380-536
[2021-01-10 01:27:41,208] {_internal.py:113} MainThread INFO in _internal:  * Detected change in '/home/user2/Desktop/DATA_PIPELINE (copy)/resources/data_pipeline_resources.py', reloading
[2021-01-10 01:27:41,365] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-10 01:27:42,596] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-10 01:27:42,597] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 339-380-536
[2021-01-10 01:28:03,896] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [10/Jan/2021 01:28:03] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-10 01:28:16,543] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [10/Jan/2021 01:28:16] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-10 01:28:33,228] {_internal.py:113} Thread-4 INFO in _internal: 127.0.0.1 - - [10/Jan/2021 01:28:33] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-10 01:29:48,810] {_internal.py:113} MainThread INFO in _internal:  * Detected change in '/home/user2/Desktop/DATA_PIPELINE (copy)/resources/data_pipeline_resources.py', reloading
[2021-01-10 01:29:49,071] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-10 01:29:50,100] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-10 01:29:50,101] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 339-380-536
[2021-01-10 01:35:23,989] {_internal.py:113} MainThread INFO in _internal:  * Detected change in '/home/user2/Desktop/DATA_PIPELINE (copy)/resources/data_pipeline_resources.py', reloading
[2021-01-10 01:35:24,131] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-10 01:35:24,950] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-10 01:35:24,951] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 339-380-536
[2021-01-10 01:40:44,468] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [10/Jan/2021 01:40:44] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-12 12:47:36,807] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 12:47:38,413] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 12:47:38,423] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 12:47:38,437] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 12:49:31,727] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 12:49:31] "[33mGET / HTTP/1.1[0m" 404 -
[2021-01-12 12:51:43,091] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 12:51:43] "[31m[1mGET /data_pipeline/dataset_validation HTTP/1.1[0m" 405 -
[2021-01-12 12:52:02,757] {_internal.py:113} Thread-4 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 12:52:02] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 12:55:17,648] {_internal.py:113} Thread-5 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 12:55:17] "[31m[1mPOST /data_pipeline/dataset_validation?dataset_type=SINGULAR&operation=SPELL_CHECKER&input_file=C:\\Users\\Dhanvi\\Spelling_errors\\test_incorrect.csv&encoding=UTF-8&language1=en HTTP/1.1[0m" 400 -
[2021-01-12 12:57:14,944] {_internal.py:113} Thread-6 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 12:57:14] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 12:57:54,541] {_internal.py:113} Thread-7 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 12:57:54] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 12:58:43,637] {_internal.py:113} Thread-8 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 12:58:43] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 12:58:56,080] {_internal.py:113} Thread-9 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 12:58:56] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 12:59:18,516] {_internal.py:113} Thread-10 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 12:59:18] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:00:53,113] {_internal.py:113} Thread-11 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:00:53] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:04:30,434] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 13:04:30,753] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:04:34,097] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:04:34,110] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:04:34,125] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 13:04:40,195] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:04:40] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:04:53,124] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 13:04:53,417] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:04:55,679] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:04:55,689] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:04:55,704] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 13:05:59,234] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\__init__.py', reloading
[2021-01-12 13:05:59,394] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:06:02,672] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:06:02,682] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:06:02,695] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 13:08:36,360] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:08:36] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:08:59,538] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\__init__.py', reloading
[2021-01-12 13:08:59,746] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:11:26,062] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:11:27,649] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:11:27,658] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:11:27,671] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 13:11:34,642] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:11:34] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:12:14,773] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 13:12:14,927] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:12:17,467] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:12:17,478] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:12:17,492] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 13:12:18,123] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:12:18] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:16:41,374] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 13:16:41,707] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:16:43,648] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:16:43,658] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:16:43,671] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 13:20:27,117] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:20:27] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:21:21,860] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:21:21] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:22:37,855] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 13:22:38,222] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:22:41,215] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:22:41,232] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:22:41,250] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 13:22:41,737] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:22:41] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:22:49,202] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 13:22:49,465] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:22:52,483] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:22:52,498] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:22:52,516] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 13:22:52,721] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:22:52] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:23:48,416] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 13:23:48,868] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:23:52,426] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:23:52,446] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:23:52,478] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 13:23:52,966] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 13:23:52] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 13:25:13,720] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\__init__.py', reloading
[2021-01-12 13:25:13,892] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 13:25:16,512] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 13:25:16,531] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 13:25:16,553] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:40:12,507] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:40:14,178] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:40:14,188] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:40:14,200] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:40:19,641] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:40:19] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 14:40:32,009] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-12 14:40:32,176] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:40:34,302] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:40:34,317] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:40:34,332] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:40:35,177] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:40:35] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 14:41:40,794] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:41:40] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 14:42:01,198] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-12 14:42:01,371] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:42:03,657] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:42:03,673] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:42:03,688] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:42:09,709] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:42:09] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 14:42:38,000] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 14:42:38,188] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:42:40,411] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:42:40,423] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:42:40,440] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:42:43,895] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:42:43] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 14:43:04,205] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-12 14:43:04,362] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:43:06,356] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:43:06,371] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:43:06,387] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:43:31,205] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 14:43:31,423] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:43:33,611] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:43:33,623] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:43:33,639] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:43:38,200] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:43:38] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 14:43:51,273] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:43:51] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 14:44:22,131] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 14:44:22,316] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:44:24,411] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:44:24,432] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:44:24,467] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:44:29,914] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:44:29] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 14:44:48,175] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 14:44:48,328] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:44:50,366] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:44:50,382] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:44:50,395] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:44:54,896] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:44:54] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-12 14:45:18,902] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 14:45:19,109] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:45:21,008] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:45:21,024] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:45:21,040] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:45:34,195] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 14:45:34,342] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:45:36,294] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:45:36,306] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:45:36,321] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:54:04,940] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 14:54:06,514] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 14:54:06,525] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 14:54:06,538] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 14:54:49,399] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:54:49] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-12 14:58:29,854] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 14:58:29] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-12 15:10:09,565] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-12 15:10:09,768] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 15:10:12,515] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 15:10:12,530] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 15:10:12,547] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 22:25:26,024] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-12 22:25:27,745] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-12 22:25:27,758] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-12 22:25:27,772] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-12 22:25:38,590] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 22:25:38] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-12 22:26:36,818] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [12/Jan/2021 22:26:36] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 11:31:32,433] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 11:31:34,832] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 11:31:34,864] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 11:31:34,888] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 11:31:40,828] {utils.py:141} Thread-2 INFO in utils: NumExpr defaulting to 4 threads.
[2021-01-13 11:31:40,948] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:31:40] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-13 11:32:18,374] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:32:18] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 11:33:59,453] {_internal.py:113} Thread-4 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:33:59] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 11:34:07,123] {_internal.py:113} Thread-5 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:34:07] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-13 11:41:08,712] {_internal.py:113} Thread-6 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:41:08] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-13 11:41:34,646] {_internal.py:113} Thread-7 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:41:34] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-13 11:45:02,708] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 11:45:03,549] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 11:45:06,738] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 11:45:06,754] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 11:45:06,782] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 11:45:11,349] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:45:11] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-13 11:45:55,673] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-13 11:45:56,066] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 11:45:59,836] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 11:45:59,851] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 11:45:59,875] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 11:46:04,799] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:46:04] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 11:47:37,123] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:47:37] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 11:47:53,328] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-13 11:47:53,593] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 11:47:57,420] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 11:47:57,436] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 11:47:57,457] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 11:49:17,720] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:49:17] "[31m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 400 -
[2021-01-13 11:49:34,218] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-13 11:49:34,514] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 11:49:37,917] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 11:49:37,933] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 11:49:37,949] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 11:49:42,513] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 11:49:42] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 13:40:01,547] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:40:01] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 13:40:33,704] {utils.py:141} Thread-4 INFO in utils: NumExpr defaulting to 4 threads.
[2021-01-13 13:40:33,754] {_internal.py:113} Thread-4 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:40:33] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 13:41:05,379] {_internal.py:113} Thread-5 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:41:05] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 13:41:12,776] {_internal.py:113} Thread-6 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:41:12] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 13:45:18,226] {_internal.py:113} Thread-7 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:45:18] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 13:45:31,800] {_internal.py:113} Thread-8 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:45:31] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 13:45:42,894] {_internal.py:113} Thread-9 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:45:42] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 13:49:54,608] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-13 13:49:55,418] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 13:50:02,476] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 13:50:02,499] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 13:50:02,523] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 13:51:46,930] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:51:46] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 13:52:26,504] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:52:26] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 13:52:38,330] {_internal.py:113} Thread-4 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:52:38] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 13:54:02,017] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-13 13:54:02,213] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 13:54:04,962] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 13:54:04,979] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 13:54:04,997] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 13:54:13,649] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-13 13:54:13,914] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 13:54:19,488] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 13:54:19,509] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 13:54:19,529] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 13:56:02,988] {utils.py:141} Thread-2 INFO in utils: NumExpr defaulting to 4 threads.
[2021-01-13 13:56:03,047] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:56:03] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 13:59:14,607] {_internal.py:113} Thread-3 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 13:59:14] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 14:06:13,492] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:06:13,896] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:06:18,161] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:06:18,179] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:06:18,207] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:07:39,487] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-13 14:07:39,759] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:07:43,129] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:07:43,153] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:07:43,207] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:24:11,820] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:24:11] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:38:15,360] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:38:15,600] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:38:17,791] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:38:17,801] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:38:17,815] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:38:23,371] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:38:23] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:39:03,442] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\resources\\data_pipeline_resources.py', reloading
[2021-01-13 14:39:03,611] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:39:05,683] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:39:05,694] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:39:05,708] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:39:11,150] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:39:11] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:40:08,246] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:40:08,530] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:40:10,843] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:40:10,856] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:40:10,872] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:40:11,759] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:40:11] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:41:38,569] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:41:38,740] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:41:57,286] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:41:59,303] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:41:59,314] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:41:59,328] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:42:03,585] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:42:03] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:42:53,012] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:42:53,199] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:42:55,330] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:42:55,342] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:42:55,356] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:43:05,346] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:43:05] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:43:39,944] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:43:40,257] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:43:42,321] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:43:42,334] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:43:42,348] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:43:49,275] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:43:49] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:44:06,234] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:44:06,401] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:44:08,869] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:44:08,887] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:44:08,906] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:44:20,110] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:44:20] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:45:27,006] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:45:27,337] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:45:29,388] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:45:29,400] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:45:29,417] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:45:32,730] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:45:32] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:45:55,973] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:45:56,145] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:45:58,520] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:45:58,535] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:45:58,548] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:47:39,355] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:47:39,532] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:47:42,541] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:47:42,552] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:47:42,565] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:47:46,617] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:47:46] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 14:49:40,953] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:49:41,129] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:49:43,741] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:49:43,756] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:49:43,773] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:49:47,140] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:49:47] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 14:50:23,433] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:50:23,618] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:50:26,184] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:50:26,199] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:50:26,215] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:50:29,249] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:50:29] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 14:50:57,738] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:50:57,909] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:51:00,337] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:51:00,353] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:51:00,368] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:51:07,297] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:51:07] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 14:56:46,530] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:56:46,688] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:57:26,883] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:57:28,825] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:57:28,836] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:57:28,850] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:57:33,150] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:57:33] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:57:47,118] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:57:47,512] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:57:50,381] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:57:50,393] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:57:50,406] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:57:53,627] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:57:53] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:58:54,581] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:58:54,834] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:58:57,271] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:58:57,283] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:58:57,298] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:59:01,045] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:59:01] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
[2021-01-13 14:59:28,982] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:59:29,190] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 14:59:31,813] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 14:59:31,828] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 14:59:31,846] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 14:59:48,118] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 14:59:48] "[35m[1mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 500 -
[2021-01-13 14:59:57,243] {_internal.py:113} MainThread INFO in _internal:  * Detected change in 'C:\\Users\\Dhanvi\\Wikipedia_Crawler\\data-validation-pipeline\\utilities\\spell_checker\\spell_corrector.py', reloading
[2021-01-13 14:59:57,480] {_internal.py:113} MainThread INFO in _internal:  * Restarting with stat
[2021-01-13 15:00:01,023] {_internal.py:113} MainThread WARNING in _internal:  * Debugger is active!
[2021-01-13 15:00:01,048] {_internal.py:113} MainThread INFO in _internal:  * Debugger PIN: 959-620-893
[2021-01-13 15:00:01,099] {_internal.py:113} Thread-1 INFO in _internal:  * Running on http://0.0.0.0:5001/ (Press CTRL+C to quit)
[2021-01-13 15:00:02,324] {_internal.py:113} Thread-2 INFO in _internal: 127.0.0.1 - - [13/Jan/2021 15:00:02] "[37mPOST /data_pipeline/dataset_validation HTTP/1.1[0m" 200 -
